CHANGES.rst

```diff
@@ -16,6 +16,8 @@ dark_current
 
 jump
 ~~~~
+-- Changes to limit the expansion of MIRI shower ellipses to be the same
```

Comment:
Need to move this entry to the top section of the change log, now that 1.2.0 is already released.